london/RESTMessageV2.d.ts

```diff
@@ -1,46 +1,511 @@
-declare namespace sn_ws {
```

Comment:
This should have been left in. See the contructor: var sm = new **sn_ws**.RESTMessageV2();

Without declaring the namespace, we don't know RESTMessageV2 is within **sn_ws**.

Reply:
maybe we can spend 2-3 mins at the office on this...